front/src/store/tv/config.ts

```diff
@@ -0,0 +1,7 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+//mock redux store cofig for dispathing action
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+export const returnWithStore = (args?: any) => mockStore(args);
```

Comment:
Could this file be moved into the tests directory?

Reply:
No this would be run as a test then and it will fail. We need to find this a central location because many tests will use this util.

Reply:
I think this should still go in the tests directory - can you try? It shouldn't run as a test because it's not `.test` or `.spec`

Reply:
sure I can try

Reply:
I am sorry this is failing with the following messages: 

![image](https://user-images.githubusercontent.com/31371040/96663658-b31a8500-1316-11eb-9021-3dce46e59c5f.png)

Reply:
I am researching jest configuration for this not to be run as a test